.github/workflows/cordformation-build-and-publish-master.yml

```
@@ -0,0 +1,34 @@
+# Build and publish for each push to branch master
+# Cordformation module
+
+name: GitHub Actions - Tagged release
```

Doesn't depend on tags so should be renamed
Please, put module name to the action name instead
```suggestion
name: GitHub Actions - Publish Cordformation Module
```